shared/blockutil/block_utils.go

```diff
@@ -23,3 +23,18 @@ func SignedBeaconBlockHeaderFromBlock(block *ethpb.SignedBeaconBlock) (*ethpb.Si
 		Signature: block.Signature,
 	}, nil
 }
+
+// BeaconBlockHeaderFromBlock function to retrieve block header from block.
+func BeaconBlockHeaderFromBlock(block *ethpb.BeaconBlock) (*ethpb.BeaconBlockHeader, error) {
```

Comment:
A basic test for this to ensure no fields missed

Reply:
added